frontend/src/stores/accounts.ts

```diff
@@ -16,7 +16,8 @@ export const invert_account = derived(
       ? (name) =>
           name.startsWith($options.name_income) ||
           name.startsWith($options.name_liabilities) ||
-          name.startsWith($options.name_equity)
+          name.startsWith($options.name_equity) ||
+          name === "Net Profit"
```

Comment:
simply compare to the translated string here `_("Net Profit")`

Reply:
D'oh! I didn't realize that the translations are available in the frontend. I'll update the PR, thanks!